npm/react/src/hooks.ts

```diff
@@ -16,22 +16,6 @@ export function setupHooks (rootId: string) {
     }
```

Comment:
Though this is not part of the fix, this clean-up is a very good idea.
I am going to check for the same issues in the vue side.